Xamarin.Forms.Platform.Tizen/ExportRendererAttribute.cs

```diff
@@ -1,12 +1,16 @@
 using System;
+using Xamarin.Forms.Core;
+using Information = Tizen.System.Information;
 
 namespace Xamarin.Forms.Platform.Tizen
 {
 	[AttributeUsage(AttributeTargets.Assembly, AllowMultiple = true)]
-	public sealed class ExportRendererAttribute : HandlerAttribute
+	public sealed class ExportRendererAttribute : BaseExportRendererAttribute
 	{
 		public ExportRendererAttribute(Type handler, Type target) : base(handler, target)
 		{
+			if(Information.TryGetValue("tizen.org/feature/platform.version", out string OSVersion))
```

Comment:
Since `TryGetValue` is being used to get various kinds of information and the result might possibly fail, should we throw an exception here in that case (i.e. if block isn't executed)? Default for `MajorVersion` is 0.

Reply:
API version would be better than platform version in this case. So, it could be `Tizen.Common.DotnetUtil.TizenAPIVersion`.  ([API Ref.](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Common.DotnetUtil.html)).

Reply:
Thanks. I can use API version.